eng/update-dependencies/VariableUpdaterBase.cs

```diff
@@ -8,19 +8,19 @@
 #nullable enable
 namespace Dotnet.Docker;
 
-internal abstract class BasicVersionUpdater : FileRegexUpdater
+internal abstract class VariableUpdaterBase : FileRegexUpdater
 {
-    protected readonly string _variableName;
-    protected readonly Lazy<JObject> _manifestVariables;
+    protected readonly string VariableName;
+    protected readonly Lazy<JObject> ManifestVariables;
```

Comment:
These are still fields. This is what I was suggesting.

```suggestion
    protected string VariableName { get; }
    protected Lazy<JObject> ManifestVariables { get; }
```